projects/project-player-library/src/lib/pages/details-page/details-page.component.ts

```diff
@@ -71,34 +75,34 @@ export class DetailsPageComponent implements OnInit {
   taskCardAction(event:any){
     switch (event.action) {
       case 'edit':
-        this.moveToTaskDetails(event._id);
+        this.moveToDetailsTask(event._id);
         break;
 
       case 'share':
         this.openDialog()
         break;
 
       case 'delete':
-        this.openDialogForDelete(event.id);
+        this.openDialogForDelete(event._id);
         break;
 
       default:
         break;
     }
   }
 
-  moveToTaskDetails(data: any) {
+  moveToDetailsTask(data: any) {
     if (!this.submitted) {
```

Comment:
why this condition and is it a generic way?

Reply:
yea